src/main/middleware/ErrorHandler.ts

```diff
@@ -14,18 +14,19 @@ export function errorHandler(
   next: NextFunction
 ): any {
   // todo: see if can figure out the extension functions
-  if (error instanceof QueryFailedError)
-    return res
-      .status(400)
-      .send(new FrontEndErrorObject(error.name, [error.message]));
   if (error instanceof ValidationError)
     return res.status(400).send(
       new FrontEndErrorObject(
         error.name,
         error.details.map((detail) => detail.message)
       )
     );
+
+  let statusCode = 400;
+
+  if (error instanceof ForbiddenError) statusCode = 403;
+
   return res
-    .status(400)
+    .status(statusCode)
     .send(new FrontEndErrorObject(error.name, [error.message]));
```

Comment:
How about this? Now we make it generic to the type contract of BaseError. Any other error is out of our control and maybe should be status 500 b/c its an unexpected internal server error
```
  if (error instanceof BaseError)
    return res.status(error.status).send(
        new FrontEndErrorObject(
            error.name,
            [error.message]
        )
    )

  return res
    .status(500)
    .send(new FrontEndErrorObject(error.name, [error.message]));
```

Reply:
Ooo!! I like this :) will update, did you want me to add back in QueryFailedError for status 400 or have it as 500 as well?

Reply:
I think 500 as well sounds good to me